modules/openapi-generator/src/main/resources/go-server/routers.mustache

```diff
@@ -170,7 +170,15 @@ func parseInt32Parameter(param string, required bool) (int32, error) {
 }
 
 // parseBoolParameter parses a string parameter to a bool
-func parseBoolParameter(param string) (bool, error) {
+func parseBoolParameter(param string, required bool) (bool, error) {
+	if param == "" {
+		if required {
+			return false, errors.New(errMsgRequiredMissing)
```

Comment:
I think would it be better to make `errors.New(errMsgRequiredMissing)` a global var in that file